modules/microshift-custom-ca-proc.adoc

```diff
@@ -0,0 +1,221 @@
+----
++
+[source,terminal]
+----
+ok
+----
+
+.. Verify the `subject` and `issuer` of your customized API server certificate authority by running the following command:
++
+[source,terminal]
+----
+$ curl --cacert /tmp/caCert.pem https://${fqdn_name}:6443/healthz -v
+----
++
+.Example output
+----
+Server certificate:
+  subject: CN=kas-test-cert_server
+  start date: Mar 12 11:39:46 2024 GMT
+  expire date: Mar 12 11:39:46 2025 GMT
+  subjectAltName: host "dhcp-1-235-3.arm.eng.rdu2.redhat.com" matched cert's "dhcp-1-235-3.arm.eng.rdu2.redhat.com"
+  issuer: CN=kas-test-cert_ca
+  SSL certificate verify ok.
+----
+
+//have this in troubleshooting, WDYT? keep here or remove?
+. Optional. Check configured certificate values against the `kube-apiserver` by running the following command:
++
+[source,terminal]
+----
+$ sudo journalctl -u microshift -b0 | grep tls-sni-cert-key
+----
++
+[Example output]
+----
+Feb 21 07:46:04 dhcp-1-235-93.arm.eng.rdu2.redhat.com microshift[1532]: kube-apiserver I0221 07:46:04.635121    1532 flags.go:64] FLAG: --tls-sni-cert-key="[/tmp/serverCert.pem
+,/tmp/serverKey.pem,/var/lib/microshift/certs/kube-apiserver-external-signer/kube-external-serving/server.crt,/var/lib/microshift/certs/kube-apiserver-external-signer/kube-external-serving/server.key;/var/lib/microshift/certs/kube-apiserver-localhost-signer/kube-apiserver-localhost-serving/server.crt,/var/lib/microshift/certs/kube-apiserver-localhost-signer/kube-apiserver-localhost-serving/server.key;/var/lib/microshift/certs/kube-apiserver-service-network-signer/kube-apiserver-service-network-serving/server.crt,/var/lib/microshift/certs/kube-apiserver-service-network-signer/kube-apiserver-service-network-serving/server.key]"
+----
+
+. Restart the {microshift-service} by running the following command:
++
+[source,terminal]
+----
+$ systemctl microshift restart
+----
+
+. Verify that the new certificate was generated by running the following command:
++
+[source,terminal]
+----
+$ oc config view --flatten
+----
++
+.Example externally generated `kubeconfig` file
++
+[source,yaml]
+----
+apiVersion: v1
+clusters:
+- cluster:
+    server: https://api.ci-ln-k0gim2b-76ef8.aws-2.ci.openshift.org:6443
+  name: ci-ln-k0gim2b-76ef8
+contexts:
+- context:
+    cluster: ci-ln-k0gim2b-76ef8
+    user:
+  name:
+current-context:
+kind: Config
+preferences: {}
+users:
+
+- name:
+  user:
+    client-certificate-data: <removed>
+    client-key-data:  <removed>
+----
+//does the microshift version have users listed in the file?
++
+[NOTE]
+====
+The `certificate-authority-data` section is not present in externally generated `kubeconfig` files.
+====
+
+. Copy the externally generated certificates you want to add to the {microshift-short} host, ensuring that the files are accessible only by the trust root of the operating system.
+
+. Configure additional CAs in the trust root of the operating system. For example, in the RHEL Client truststore on the client system.
```

Comment:
[link for configuring CAs in rhel trust store](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/using-shared-system-certificates_securing-networks):